src/ec/packages/ec-component-site-header-harmonised/site-header-harmonised.story.js.bckp

```diff
@@ -45,4 +47,19 @@ storiesOf('Components/Site Headers/Harmonised', module)
     {
       notes: { markdown: notes, json: dataGroup2 },
     }
+  )
+  .add(
+    'group 3',
+    () =>
+      siteHeaderHarmonised(
+        merge(dataGroup3, {
+          logo: {
+            src:
+              'https://inno-ecl.s3.amazonaws.com/media/examples/placeholder.svg',
```

Comment:
This should come from the adapter, the things we do in the stories are not retrieved by php-storybook (the one you start with start:php-ec), so better working in the specs or in the adapter when it's possible.